radareorg/cutter
# Update hex side panel only when visible.

**Detailed description**

Don't perform unnecessary  side panel updates.

**Test plan (required)**

1) Select something 
2) close side panel, open panel
3) observe that content is correct
4) close panel, remove selection, open panel
5) observe that content is correct
6) close panel, change selection, open panel
7) observe that content is correct
8) do steps 1-7 replacing close open panel with tab switching

**Closing issues**

Closes #1528
